src/test/scala/com/microsoft/hyperspace/index/CreateIndexNestedTest.scala

```diff
@@ -160,25 +160,23 @@ class CreateIndexNestedTest extends HyperspaceSuite with SQLHelper {
       // should be added to index schema if they are not already among index config columns.
       assert(
         indexRecordsDF.schema.fieldNames.sorted ===
-          (SchemaUtils.prefixNestedFieldNames(
-            indexConfig2.indexedColumns.zip(Seq(true)) ++
-            indexConfig2.includedColumns.zip(Seq(true))) ++
+          ((indexConfig2.indexedColumns ++ indexConfig2.includedColumns).map(
+            ResolvedColumn(_, isNested = true).normalizedName) ++
             Seq(IndexConstants.DATA_FILE_NAME_ID) ++ partitionKeys).sorted)
     }
   }
 
   test("Check lineage in index records for non-partitioned data.") {
     withSQLConf(IndexConstants.INDEX_LINEAGE_ENABLED -> "true") {
-      hyperspace.createIndex(nonPartitionedDataDF, indexConfig1)
+      hyperspace.createIndex(nonPartitionedDataDF, indexConfig2)
```

Comment:
`ResolvedColumn` has an assert that if we are setting `isNested = true`, the column name should contain dots. `indexConfig1` has names both with and without dots, whereas `indexConfig2` only has names with dots. So I changed it to use `indexConfig2` (also same as the test right above - `Check lineage in index records for partitioned data when partition key is not in config.`). Since it's testing if the lineage column is included, it should be OK to change it to `indexConfig2`.

Reply:
Shouldn't we test if the functionality work correct with both types of column names in one of the `indexedColumns` or `includedColumns`? The `indexConfig1` does contain `Seq("Date", "nested.leaf.cnt")` in the included columns property while `indexConfig2` doesn't have this mixed column types. We should have such tests that verifies that this mixed cased does work.

Reply:
`Index creation with nested indexed and included columns.` should already handle `indexConfig1`? (I can update it make the coverage better)